docs/standard/components.md

```diff
@@ -0,0 +1,81 @@
+---
+title: .NET Components
+description: Describes key .NET concepts such as the .NET Standard Library, .NET runtimes, tooling, and how they interplay.
+keywords: .NET, .NET Standard Library, .NET Standard, .NET Core, .NET Framework, Xamarin, MSBuild, C#, F#, VB, compilers
+author: cartermp
+manager: wpickett
+ms.author: cartermp
+ms.date: 11/16/2016
+ms.topic: article
+ms.prod: .net
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 2e38e9d9-8284-46ee-a15f-199adc4f26f4
+---
+
+# .NET Components
+
+.NET is made up of a number of components.  It has a standard library, called the .NET Standard Library, which is a large set of APIs which runs everywhere.  This standard library is implemented by three .NET runtimes - .NET Framework, .NET Core, and Mono for Xamarin.  The .NET languages also run on any .NET runtime.  Additionally, there are tools on every platform which allow you to build projects.  These tools are the same regardless of your choice of runtime.
+
+Here's a graphical overview of each of the "pieces" of .NET and how they fit.
+
+![All .NET Concepts Together](media/dotnet-things.png)
+
+What follows is a brief explanation of each of the key concepts shown above.  
+
+## .NET Standard Library
+
+The .NET Standard Library is a set of APIs which are implemented by a .NET runtime.
+
+More formally, it is a specification of .NET APIs which make up a uniform set of contracts that you compile your code against.  These contracts have underlying implementations in a given .NET runtime.  This enables portability across different .NET runtimes, making it so that your code can effectively "run everywhere".
+
+The .NET Standard Library is also a build target, where it is known as the .NET Standard.  You can target .NET Standard 1.0-1.6 right now, and will be able to target .NET Standard 2.0+ in the future.  If your code targets a version of the .NET Standard, it is guaranteed to run on any .NET runtime which implements that version.
+
+To learn more about the .NET Standard library and targeting the .NET Standard, check out [.NET Standard Library](library.md).
+
+## .NET runtimes
+
+There are three primary .NET runtimes which Microsoft actively develops and maintains: .NET Core, .NET Framework, and Mono for Xamarin.
```

Comment:
In this case, no, because the version of Mono Xamarin uses isn't quite the same as the "true" Mono.